ui/lib/config-ui/addon/components/messages/page/create-and-edit-message-form.hbs

```diff
@@ -76,4 +83,29 @@
       />
     </Hds::ButtonSet>
   </div>
-</form>
\ No newline at end of file
+</form>
+
+{{#if this.showMessagePreviewModal}}
+  {{#if (eq @message.type "modal")}}
+    <Hds::Modal
+      id="message-modal-preview"
+      @size="large"
+      @color="warning"
+      @onClose={{fn (mut this.showMessagePreviewModal) false}}
+      data-test-modal="preview modal"
+      as |M|
+    >
+      <M.Header data-test-modal-title={{@message.title}}>
+        {{@message.title}}
+      </M.Header>
+      <M.Body data-test-modal-body>
+        {{@message.message}}
+      </M.Body>
+      <M.Footer as |F|>
+        <Hds::Button @text="Confirm" {{on "click" F.close}} data-test-modal-button="Close" />
+      </M.Footer>
+    </Hds::Modal>
+  {{else}}
+    <Messages::PreviewImage @message={{@message}} @showMessagePreviewModal={{this.showMessagePreviewModal}} />
```

Comment:
Do we need this arg? It will always be true since this whole block doesn't render unless `this.showMessagePreviewModal` is true, right?

Reply:
Ah I see it's edited in the child component to hide/show the modal. Should this variable be added as a tracked property to the component file?